storage/statedb/cache_fastcache.go

```diff
@@ -41,23 +43,27 @@ type FastCache struct {
 }
 
 // newFastCache creates a FastCache with given cache size.
-// If you want auto-scaled cache size, set config.LocalCacheSizeMB to AutoScaling.
+// If you want auto-scaled cache size, set config.LocalCacheSizeMiB to AutoScaling.
 // It returns nil if the cache size is zero.
 func newFastCache(config *TrieNodeCacheConfig) TrieNodeCache {
-	if config.LocalCacheSizeMB == AutoScaling {
-		config.LocalCacheSizeMB = getTrieNodeCacheSizeMB()
+	if config.LocalCacheSizeMiB == AutoScaling {
+		config.LocalCacheSizeMiB = getTrieNodeCacheSizeMiB()
 	}
 
-	if config.LocalCacheSizeMB <= 0 {
+	if config.LocalCacheSizeMiB <= 0 {
 		return nil
 	}
 
-	fc := &FastCache{fast: fastcache.LoadFromFileOrNew(config.FastCacheFileDir, config.LocalCacheSizeMB*1024*1024)} // Convert MB to Byte
+	logger.Info("Initializing local trie node cache (fastCache)",
+		"MaxMiB", config.LocalCacheSizeMiB, "FilePath", config.FastCacheFileDir)
```

Comment:
Is there a need for the new log?

Reply:
Yes It is. It takes more than 15 sec on my mac mini.
```
INFO[02/22,09:17:48 +09] [5] Using DeriveShaConcat!
INFO[02/22,09:17:48 +09] [47] Initializing local trie node cache (fastCache)  MaxMiB=7000 FilePath=/Volumes/Samsung_T5/cypress_migrated/datadir/fastcache
INFO[02/22,09:18:04 +09] [47] Initialized local trie node cache (fastCache)  LoadedMiB=7008 LoadedEntries=20434023 elapsed=15.862882951s
```